tests/cycpp_tests.py

```diff
@@ -585,16 +585,19 @@ def test_schemafilter():
         '    "<interleave>\\n"\n'
         '    "    <element name=\\"x\\">\\n"\n'
         '    "        <oneOrMore>\\n"\n'
-        '    "            <element name=\\"key\\">\\n"\n'
-        '    "                <data type=\\"int\\"/>\\n"\n'
-        '    "            </element>\\n"\n'
-        '    "            <element name=\\"val\\">\\n"\n'
-        '    "                <data type=\\"double\\"/>\\n"\n'
+        '    "            <element name=\\"item\\">\\n"\n'
+        '    "                <interleave>\\n"\n'
```

Comment:
Is this interleave important?  It seems that the interleave at this level would allow someone to create a file that would visually appear as value=key rather than key=value.  Is there a detail of JSON that requires this?